WordPress/Classes/Utility/Migrations/BlogToBlogSettings40to41.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+
+
+/**
+ *  @class      BlogToBlogSettings40to41
+ *  @brief      This class should be executed during the migration from Data Model Mark 40 >> 41.
+ *              It's main goal is to set the relationship between the *Blog* and *BlogSetting* instances,
+ *              in the destination context.
+ */
+class BlogToBlogSettings40to41: NSEntityMigrationPolicy
```

Comment:
It turns out, you don't even need this. You can set this as the Value Expression for the `blog` relationship in `BlogToBlogSettings` and remove your custom policy:

```FUNCTION($manager, "destinationInstancesForEntityMappingNamed:sourceInstances:" , "BlogToBlog", $source)```

![screen shot 2015-11-16 at 11 44 18](https://cloud.githubusercontent.com/assets/8739/11179974/7093e820-8c57-11e5-8b64-6415de1f9a0b.png)

The migration test passes, but I haven't tested manually